pkgs/applications/science/meteorology/supercell-wx/default.nix

```diff
@@ -0,0 +1,158 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  aws-sdk-cpp,
+  bzip2,
+  cmake,
+  ninja,
+  zlib,
+  openssl,
+  curl,
+  glew,
+  geos,
+  git,
+  boost185,
+  spdlog,
+  stb,
+  libcpr_1_10_5,
+  libpng,
+  libSM,
+  geographiclib,
+  howard-hinnant-date,
+  re2,
+  gtest,
+  glm,
+  qtbase,
+  qttools,
+  qtmultimedia,
+  qtpositioning,
+  qtimageformats,
+  tbb_2021_11,
+  tzdata,
+  substituteAll,
+  python3,
+  wrapQtAppsHook,
+}:
+let
```

Comment:
I think it is preferred to have these in the derivation and use the `finalAttrs` pattern, to allow overriding.

Reply:
I assume you mean `src`/`version` below these lines. I have made it so.